src/logging.rs

```diff
@@ -54,13 +56,11 @@ macro_rules! infoheader {
 	}};
 }
 
-#[macro_export]
 macro_rules! infoentry {
-	($str:expr, $rhs:expr) => ($crate::infoentry!($str, "{}", $rhs));
+	($str:expr, $rhs:expr) => (infoentry!($str, "{}", $rhs));
```

Comment:
Macro hygiene:

```suggestion
	($str:expr, $rhs:expr) => ($crate::infoentry!($str, "{}", $rhs));
```

Reply:
I don't export infoentry. Consequently, the compiler don't find `$crate::infoentry`

Reply:
Yeah, I noticed as well: https://github.com/hermitcore/libhermit-rs/pull/323#issuecomment-1003723286